content/02.main-text.md

```diff
@@ -341,7 +341,7 @@ Although these criteria provided a straightforward, equitable way to determine w
 In biomedical journals, the convention is that the first and last authors made the most substantial contributions to the manuscript.
 This convention can be difficult to reconcile in a collaborative effort.
 Using Git, we could quantify the number of commits each author made or the number of sentences an author wrote or edited, but these metrics discount intellectual contributions such as discussing primary literature and reviewing pull requests.
-Therefore, we concluded that it is not possible to construct an objective system to compare and weight the different types of contributions and produce an ordered author list.
+Therefore, we concluded that it is not possible to construct an objective system to compare and weight the different types of contributions and produce an ordered author list [@url:http://ivory.idyll.org/blog/2019-authorship-revisiting.html].
```

Comment:
@agitter I think this citation fits here and should be included somewhere in the manuscript because it's about authorship for open collaborative projects / manuscripts